src/components/switch/content/Switch.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { SwitchProps } from './switch_types';
+import { Label, ToggleButton, Slider, Wrapper } from '../styles/Switch.styles';
+
+const Switch: React.FC<SwitchProps> = ({ ...rest }: SwitchProps) => {
+  const [checked, setChecked] = React.useState<boolean>(rest.checked);
```

Comment:
mostaghim props ro nakon too state ghalate in kar

Reply:
khb age kar bar checked= true bezne bayad dokmeye ma roshan she hata ba refresh chejuri handle konam?

Reply:
in kari ke gofti kardam kar nakard